staff/edgar-frias-jimenez/cv/index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My CV</title>
+  </head>
+  <body>
+    <main>
+      <header>
+        <h1>CV First Name</h1>
+        <left>
```

Comment:
@edfrias `<left>`and `<right>`are not valid html semantic tags. please replace them for valid ones. thanks.
<img width="1338" alt="Screenshot 2019-04-03 at 15 56 23" src="https://user-images.githubusercontent.com/44497453/55484658-32f3d800-5629-11e9-8689-ecfbb7347b98.png">

Reply:
@lexbarba done, thanks!